spec/factories/games.rb

```diff
@@ -5,4 +5,8 @@
     association :black_player, factory: :user
     state "open"
   end
+  
+  factory :invalid_game, parent: :game do 
```

Comment:
FYI: If you nest the child factory inside the do/end block used to create the parent, then you don't have to specify the parent explicitly using the `parent:` option.